app/models/doctor.rb

```diff
@@ -1,6 +1,6 @@
 class Doctor < ApplicationRecord
   has_many :reservations
-  has_many :users, through: :reservations
+  has_belongs_to :user
```

Comment:
Hello @samiullahbahadur , good job so far as you tried to implement the changes, but there are some issues that you need to work on: 
 I think you need to fix this line because this association does not exist.
You should use either `has_many` or `has_many_and_belongs-to`  with two tables

Reply:
okay you right, i have fix it please check now